.github/workflows/build.yaml

```diff
@@ -98,7 +98,7 @@ jobs:
 
       - name: Install dependencies
         run: |
-          pip install --upgrade setuptools wheel cibuildwheel
+          pip install --upgrade "setuptools<65" "flake8<5" wheel cibuildwheel
```

Comment:
Remove flake8 pin?

Reply:
Without it fails as it run `pytest` in the workflow.

Reply:
Sorry wrong line.